styles/main.css

```diff
@@ -4,23 +4,24 @@ body {
     font-family: 'Big Shoulders Display', cursive;
 }
 h1 {
-    color: #801515;
+    color: transparent;
     
     background-repeat: repeat-x;
     font-family: 'Mansalva', cursive;
     text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.2);
     background: -webkit-linear-gradient(#eee, #097921);
     -webkit-background-clip: text;
-    -webkit-text-fill-color: transparent; 
+    background-clip: text;

```

Comment:
Just use the regular `color` property to set transparent color.
Always include the prefix-less version of a property as well.